base/src/dispatcher/dispatcher.ts

```diff
@@ -732,7 +732,7 @@ export class Dispatcher implements ZLUX.Dispatcher {
   private createAsync(plugin:ZLUX.Plugin, action:Action, eventContext: any):Promise<ActionTarget>{
     //let appPromise:Promise<MVDHosting.InstanceId>
     if (!this.launchCallback){
-      return Promise.reject("no launch callback established");
+      return Promise.reject("ZWED5030E - No launch callback established");
```

Comment:
Same comment as anything involving 'throw'.  These messages are also included in the resource file containing all the message ID's, but if we are just manually printing the message it seems like a waste of space to also include it in the resource file.

Reply:
Messages are in the resource file for future use, when we have a mechanism to retrieve and use them. Not adding them to the file would cause extra work and coordination challenges at a later time. We are implementing as much of the final solution as possible now.